tests/e2e/features/artifacts/depositPage/artifacts-deposits.feature

```diff
@@ -60,4 +60,34 @@ Feature: Artifacts - UI
     Then Element with "testId" "fee-amount" should be "visible"
     Then Fee should have "$" value
     Then Fee should have "ETH" value
-    Then Element with "text" " Continue " should be "visible"
\ No newline at end of file
+    Then Element with "text" " Continue " should be "visible"
+
+  @id1647 @tokens
+  Scenario Outline:  Check search functionality for Choose Tokens (with results)
+    Given I go to page "<network>"
+    When I click by "testId" with "your-account" value
+    Then I click by "testId" with "token-dropDown" value
+    When I fill the "//*[@placeholder='Symbol or address']" input field by "USDC"
+    Then Element with "text" "<token name>" should be "visible"
+    Then Element with "text" "<token address>" should be "visible"
+    Then Element with "class" "token-balance-amount" should be "visible"
+
+    Examples:
+      | network                                              | token name | token address  |
+      | /transaction/zksync/era/deposit/?network=era-mainnet | USDC       | 0x3355df...af4 |
+      | /transaction/zksync/era/deposit/?network=era-goerli  | USDC       | 0x0faF6d...2A9 |
+
+
+  @id1548 @tokens
+  Scenario Outline:  Check search functionality for Choose Tokens (no results)
+    Given I go to page "<network>"
+    When I click by "testId" with "your-account" value
+    Then I click by "testId" with "token-dropDown" value
+    When I fill the "//*[@placeholder='Symbol or address']" input field by "AAA"
```

Comment:
@abilevych are we going to make a human readable records in tests by changing xpath to some simple string record or this is ok for us?